Utils/layouts/index.js

```diff
@@ -7,6 +7,7 @@ import {
   import styled from "styled-components";
   import { useRouter } from "next/router";
   import { withTranslation, Link } from "../translation/i18n";
+  import titleBackground from "../../public/assets/Images/title_bg_img.png"
```

Comment:
Could we use a background image that more closely matches the design? It's also a fair bit larger than it needs to be since it is a repeated pattern.